src/applications/representatives/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
-  "appName": "Representatives",
+  "appName": "Accredited Representative Portal",
   "entryFile": "./app-entry.jsx",
-  "entryName": "representatives",
-  "rootUrl": "/representatives",
+  "entryName": "representative",
+  "rootUrl": "/representative",
```

Comment:
Root url = `/accredited-representative-portal`?

Reply:
If we can figure out if it's possible, we could change to `"rootUrl": "/"`. Someone would have to think through that. How would you get that to work on localhost and other environments? Here are the [docs around routing](https://depo-platform-documentation.scrollhelp.site/developer-docs/frontend-architecture#Frontendarchitecture-RoutingforReactapps).